frontend/src/components/layout/footer/FooterController.jsx

```diff
@@ -1,7 +1,23 @@
-import FooterView from "./FooterView"
-
-const FooterController = () => (
-    <FooterView />
-)
+import React from 'react'
```

Comment:
It would be cleaner to do:

```js
import { useState } from 'react'
```

and use it directly instead of React.useState

Reply:
Fixed